arbitrum-docs/partials/_troubleshooting-nodes-partial.md

```diff
@@ -68,15 +68,42 @@
 
 
 ### How can I verify that my node is fully synced? {#how-can-i-verify-that-my-node-is-fully-synced}
-<p>You can make an <code>eth_syncing</code> RPC call to your node. If it returns false, it means that the node is fully synced. If it returns an object of information, it means it is still syncing. Here's how you can interpret that information:</p>
+<p>You can make an <code>eth_syncing</code> RPC call to your node. When a nitro node is fully synced, <code>eth_syncing</code> returns the value <code>false</code> (just like a normal Geth node).</p>
+
+<p>When a nitro node is still syncing, <code>eth_syncing</code> returns a map of values to help understand why the node is not synced. Nitro execution and bottleneck are different from a normal Geth node, so <code>eth_syncing</code> output is unique to nitro. Here, we provide more details:</p>
+
+<p></p>
+
+<p><strong>Introduction: Messages, Batches, and Blocks:</strong></p>
```

Comment:
Should we remove the word "Introduction" from here? I wouldn't consider this an introduction, just a description of Messages, Batches and Blocks.

Reply:
How about "Messages, Batches and Blocks"?